include/seastar/rpc/rpc.hh

```diff
@@ -348,6 +348,9 @@ public:
     connection_id get_connection_id() const noexcept {
         return _id;
     }
+    stats& get_stats_internal() noexcept {
```

Comment:
thought it was a private, but it's actually a `public`...

Reply:
It's for `wait_for_reply` templatized helper

Reply:
i see. no wonder it's would be more convoluted if we wanted to mark it as a `friend`.